lua/snippets/inserters/floaty.lua

```diff
@@ -124,20 +124,21 @@ local function entrypoint(structure)
 	local start_win = api.nvim_get_current_win()
 	local start_buf = api.nvim_get_current_buf()
 	local start_pos = api.nvim_win_get_cursor(start_win)
+	local start_win_pos = api.nvim_win_get_position(start_win)
 	local preview_buf, preview_win, preview_opts = floaty_popup {
 		relative = 'win';
 		win = start_win;
 		bufpos = {start_pos[1]-1, start_pos[2]};
-		col = 0;
-		row = 0;
+		col = start_win_pos[2];
+		row = api.nvim_win_get_config(start_win).relative ~= "" and start_win_pos[1] or 0;
```

Comment:
This is clunky and may not work with all configs. I'm missing something I think.

The problem this "fixes" is that if I just use `start_win_pos[1]`, in the normal editor window the `floaty_popup` is one line too low (because if you run `api.nvim_win_get_position` for the normal window it returns (at least for me) `{ 1, 0 }`, and that row should be zero for `floaty_popup`).